### PR TITLE
Fix Suzana's bio, already updated on meetup and connpass

### PR DESCRIPTION
**Pull request by lunardog**
This PR is just a formality, already updated in public sites, since the event is coming up soon.

### CODE REVIEW

**Comment by lunardog** [comment]
one approve plz 👍 お願い

**Comment by lunardog** [comment]
@vanhuyz updated the title.

File: _posts/2018-04-12-Learn the ABC of NLP.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Wait... a Machine Learning event at Cookpad? The famous recipe sharing service c
 - Lightning talks:
   - [Alameen Najjar](https://jp.linkedin.com/in/alameen-najjar) *10 things you need to know before you apply machine learning to satellite images*
   - [Van Huy](https://github.com/vanhuyz) *3 Challenges in Customer Feedback Classification*
+  - [Chie Hayashida](#) *TBD*
```
[the remark this answers is not in the document]
Comment (vanhuyz) [reply _posts/2018-04-12-Learn the ABC of NLP.md]: @lunardog and a link to her profile?

**Comment by chie8842** [comment]
@vanhuyz @lunardog Thank you for reviewing and patch the Website. :pray:

**Pull request merged**